src/AzureClient/AzureClient.cs

```diff
@@ -1,24 +1,249 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
 #nullable enable
 
 using System;
 using System.Collections.Generic;
-using System.Text;
-using Microsoft.Jupyter.Core;
+using System.Linq;
+using System.IO;
 using System.Threading.Tasks;
-
+using Microsoft.Azure.Quantum.Client;
 using Microsoft.Identity.Client;
 using Microsoft.Identity.Client.Extensions.Msal;
-using System.Linq;
-using System.IO;
-using Microsoft.Quantum.Runtime;
+using Microsoft.Jupyter.Core;
+using Microsoft.Quantum.Simulation.Core;
 
 namespace Microsoft.Quantum.IQSharp.AzureClient
 {
     /// <inheritdoc/>
     public class AzureClient : IAzureClient
     {
+        private string ConnectionString { get; set; }
+        private string ActiveTargetName { get; set; }
+        private AuthenticationResult? AuthenticationResult { get; set; }
+        private IQuantumClient? QuantumClient { get; set; }
+        private Azure.Quantum.Workspace? ActiveWorkspace { get; set; }
+
+        /// <summary>
+        /// Creates an AzureClient object.
+        /// </summary>
+        public AzureClient()
+        {
+            ConnectionString = string.Empty;
```

Comment:
Would it make sense to use an initializer on each of L22 and L23 instead to avoid needing an explicit constructor here? I left a suggestion above to that effect.

Reply:
I learn something new about C# every day...

---
In reply to: [427460618](https://github.com/microsoft/iqsharp/pull/144#discussion_r427460618) [](ancestors = 427460618)